cfgov/jinja2/v1/_layouts/base.html

```diff
@@ -60,9 +60,7 @@
     <meta name="description"
           content="
             {%- block desc -%}
-                {{- page.search_description if page and page.search_description
-                    else 'Our vision is a consumer finance marketplace that works for American consumers, responsible providers, and the economy as a whole.'
-                -}}
+                {{- meta_description -}}
```

Comment:
The new `meta_description` will only be available to Wagtail pages, and this template can serve plain Django pages, as well, so we'll need to maintain a check for `page` and offer a fallback.

```suggestion
                {{- meta_description if page else '' -}}
```